exercises/practice/hello-world/.meta/Example.bat

```diff
@@ -0,0 +1 @@
+@echo Hello, World!
```

Comment:
Not that it matters to a  case insensitive system, but should the file name be lowercase to adhere to style used.  Also, just because the MS-DOS file system is case insensitive does not mean the files themselves will be stored there, and it also does not mean that the tooling that is being used to find and copy and remove this in a temporary file system (for CI/testing) will be case insensitive.

This is already merged, and it may end up not being a problem, just an area where I would keep my eyes on, in case we end up with strange or  unexpected errors later.

Reply:
It's because I saw like this in java's Hello World. So I thinked if exercism's structure like this, i can use this. So I don't chice this way.

Reply:
Oh then I would suggest to use lowercase. It is completely up to the track to choose the casing, basically the track should use what is commonly used in the language. Is that lowercase?

Reply:
https://github.com/exercism/batch/pull/3#discussion_r1445937657

Summary: I'll write an 'best practice' of batch and I'll update all batch files.